cms/djangoapps/contentstore/management/commands/tests/test_git_export.py

```diff
@@ -74,27 +74,15 @@ def test_error_output(self):
         """
         Verify that error output is actually resolved as the correct string
         """
-        output = StringIO.StringIO()
         with self.assertRaisesRegexp(CommandError, unicode(GitExportError.BAD_COURSE)):
             call_command(
-                'git_export', 'foo/bar:baz', 'silly',
-                stdout=output, stderr=output
+                'git_export', 'foo/bar:baz', 'silly'
             )
-        self.assertIn('Bad course location provided', output.getvalue())
-        output.close()
```

Comment:
I'm confused as to why we would want remove the assert test and the stdout and stderr. If we are not verifying the output is the test valid?